Sources/BraveWallet/Crypto/Portfolio/AddCustomAssetView.swift

```diff
@@ -16,10 +16,30 @@ struct AddCustomAssetView: View {
   
   @ObservedObject private var networkSelectionStore: NetworkSelectionStore
 
+  enum TokenType: Int, Identifiable {
+    case token
+    case nft
+    
+    var id: Int {
+      return rawValue
+    }
+    
+    var title: String {
+      switch self {
+      case .token:
+        return Strings.Wallet.addCustomTokenTitle
+      case .nft:
+        return Strings.Wallet.addCustomNFTTitle
+      }
+    }
+  }
+  
+  @State private var tokenType: Int = TokenType.token.id
```

Comment:
I think we scan tore this as the `TokenType` enum instead of an `Int`? Then use `switch` statements instead of `tokenType == TokenType.nft.rawValue` throughout?

Reply:
fixed in [299d530](https://github.com/brave/brave-ios/pull/6360/commits/299d530805abcc2e35a9cce53897e5f42c774c62)